index.js

```diff
@@ -46,9 +46,9 @@ function checkOsaScript() {
 }
 
 function checkWmic() {
-	return execa.stdout('wmic', ['useraccount', 'where', 'name="%username%"', 'get', 'fullname'])
+	return execa.stdout('wmic', ['useraccount', 'where', 'name="%username%"', 'get', 'fullname'], {shell: true})
```

Comment:
I don't want to use `shell`. It comes with a lot of downsides. Can you use `process.env.USERNAME` instead?